lib/types/db.ts

```diff
@@ -0,0 +1,53 @@
+import Sequelize, { DataTypeAbstract, DefineAttributeColumnOptions } from 'sequelize';
+import { db } from './index';
+
+export type SequelizeAttributes<T extends { [key: string]: any }> = {
+  [P in keyof T]: string | DataTypeAbstract | DefineAttributeColumnOptions
+};
+
+export type CurrencyFactory = {
+  id: string;
+};
+
+export type CurrencyAttributes = CurrencyFactory;
+
+export type CurrencyInstance = CurrencyAttributes & Sequelize.Instance<CurrencyAttributes>;
+
+export type PeerFactory = {
+  address: string;
+  port: string;
```

Comment:
One more thing I found here on a second pass, everywhere else that we have a `port` property it's a `number`. We should change this to use `number` to stay consistent.

Reply:
Thanks, i'll fix it